ci/Jenkinsfile-kubernetes

```diff
@@ -165,13 +165,68 @@ def testRInternalBackend(registryId, master, version) {
         sh "kubectl get pod driver-r | grep -q Completed && echo \"OK\" || exit 1"
 }
 
+def installHeadlessService() {
```

Comment:
nit: maybe I would rename it to `installSparkHeadlessService`, since there will be H2o headless service for external backend.

Reply:
Good point!